src/AggregateChanged.php

```diff
@@ -65,32 +53,23 @@ public function aggregateId()
      * The payload should only contain scalar types and sub arrays.
      * The payload is normally passed to json_encode to persist the message or
      * push it into a message queue.
-     *
-     * @return array
      */
-    public function payload()
+    public function payload(): array
     {
         return $this->payload;
     }
 
-    /**
-     * @param string $aggregateId
-     */
-    protected function setAggregateId($aggregateId)
+    protected function setAggregateId(string $aggregateId): void
     {
-        Assertion::string($aggregateId);
         Assertion::notEmpty($aggregateId);
```

Comment:
This line should be replaced, since the "beberlei/assert" dependency is not available in composer anymore.